README.md

```diff
@@ -4,17 +4,32 @@
 pip-custom-platform
 ===================
 
-pip+wheel wrapper which allows you to choose a custom platform name for
-building, downloading, and installing wheels.
+[pip][pip]+[wheel][wheel] wrapper which allows you to choose a custom platform
+name for building, downloading, and installing wheels.
 
-This package assumes you're running your own pypi server and would like
+This package assumes you're running your own PyPI server and would like
 support for wheels on named platforms that would otherwise be considered
 equivalent by the wheel infrastructure (for example not all linux_x86_64 are
 created equal).
 
-pip: https://github.com/pypa/pip
+## Default platform names
 
-wheel: https://bitbucket.org/pypa/wheel
+By default, pip-custom-platform guesses a platform name for you based on the
+`platform` module for Linux, and uses the default platform name on Windows, OS
+X, or other systems. Some examples:
+
+| Platform                | Default Platform Name      |
+|-------------------------|----------------------------|
+| Ubuntu Trusty (14.04)   | linux_ubuntu_trusty_x86_64 |
+| Debian Jessie (8)       | linux_debian_8_x86_64      |
+| CentOS 7                | linux_centos_7_x86_64      |
+| Fedora 22               | linux_fedora_22_x86_64     |
+| Red Hat 7               | linux_rhel_7_x86_64        |
+| openSUSE 13.2           | linux_opensuse_13_x86_64   |
+| Arch Linux              | linux_arch_unknown_x86_64  |
```

Comment:
IMO arch is one of those distros where wheels are nonsense.
They have a single rolling version if I understand it, so it's "unknown" and completely undefined whether any prior wheel will work today.